nemo/collections/asr/data/audio_to_text.py

```diff
@@ -11,17 +11,22 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import io
+import os
 from typing import Callable, Dict, List, Optional, Union
 
+import braceexpand
 import torch
+import webdataset as wd
 
+from nemo import logging
```

Comment:
It is now `from nemo.utils import logging`

Reply:
Updated